src/lua-function-list.json

```diff
@@ -422,6 +422,7 @@
     "sendATCP": "sendATCP(message, what)",
     "sendGMCP": "sendGMCP(command)",
     "sendIrc": "sendIrc(target, message)",
+    "openIRC": "openIRC()",
```

Comment:
Didn't we have some automation for this file, so no manual edits required? Or is this currently offline?

Reply:
This is automated, so indeed doesn't need to be filled in.